Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM quay.io/ncigdc/apache-base:2.4.18-1.0.0 as build
+FROM quay.io/ncigdc/apache-base:python3 as build
 
 
 COPY . /indexd
 WORKDIR /indexd
 
 RUN apt-get update \
  && apt-get install -y --no-install-recommends \
-    python2.7 \
-    python-dev \
-    python-pip \
-    python-setuptools \
+    python3 \
+    python3-dev \
+    python3-pip \
+    python3-setuptools \
     libpq-dev \
     libpq5 \
     gcc \
- && pip install wheel \
- && pip install -r build/requirements.txt \
- && python setup.py install 
+ && pip3 install wheel \
+ && pip3 install -r build/requirements.txt \
+ && python3 setup.py install
 
-FROM quay.io/ncigdc/apache-base:2.4.18-1.0.0
+FROM quay.io/ncigdc/apache-base:python3
```

Comment:
Is this the final tag going forward, or should we wait on merging this until it receives a numbered tag?

Reply:
Yeah, we should make an apache-base:1.0.2-python3 or something

Reply:
Tag will be `1.0.2-py3` once https://github.com/NCI-GDC/apache-base/pull/8 is merged and tagged.

Reply:
I know we changed the top FROM directive, but do we need to change this one as well?

Reply:
> I know we changed the top FROM directive, but do we need to change this one as well?

Yep, the bottom `FROM` directive is what goes into the final image.